var/spack/repos/builtin/packages/sqlite/package.py

```diff
@@ -4,6 +4,7 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 import os
 import re
+import sys
 from tempfile import NamedTemporaryFile
 
 import spack.platforms
```

Comment:
It just occurred to me: `sqlite` is an `AutotoolsPackage`, and I thought that class defined a conflict with `platform=windows`. Is `sqlite` needed as a dependency of `hdf5`? If not, then we could move it to another PR. If you want to keep it, I would want to make sure we can install/concretize it. We could leverage multi-build-system support to make it easier to install this on Windows in that case.

Reply:
I'll be phasing the work on sqlite out in my next commit; stay tuned

Reply:
Can you repurpose those commits into a separate PR? Sqlite support is something I'm very interested in (and assumed this PR would cover) and if you don't have the cycles to do the extra work, I can/will happily take over/add it to my mega package PR.